frameworks/Haskell/servant/hasql/src/ServantBench.hs

```diff
@@ -148,6 +164,32 @@ multipleDb pool gen mQueryId = do
 
 -- * Test 4: Fortunes
 
+type (>) a b = a Html.> b
```

Comment:
You can also, I think, add `type (>), type (#)` to the unqualified `Html` import line, I think.

Reply:
I tried; but it doesn't work as the `>` I want here is a data type. I'd have to perform an unbound unqualified import of the `Html` library to access it otherwise. IMO having multiple unbound imports in a module makes it very confusing where a function/type comes from.

Reply:
Fixed in: https://github.com/TechEmpower/FrameworkBenchmarks/pull/4553/commits/8d4d5c42f42a3892b63dd38f17152862c0491377